ui/src/app/kvm/views/KVMDetails/KVMSummary/KVMSummary.tsx

```diff
@@ -27,8 +27,18 @@ const KVMSummary = (): JSX.Element => {
   }, [dispatch]);
 
   if (!!pod) {
+    // Fetch the IP address that is contained in the power_address. This is the
+    // only place the IP address is exposed.
+    const ip_address = pod.power_address.match(/[\d.]+/)[0];
```

Comment:
Was it decided to only show the ip portion of a power address now? The legacy version just shows the whole power address.

Reply:
Ah right, was just going by the doc which has:
> IP address URL: copyable ip address for the instance.

@amylily1011 should this just replicate what already exists? Examples below:

<img width="615" alt="Screen Shot 2020-08-20 at 9 27 00 am" src="https://user-images.githubusercontent.com/361637/90699788-879ffd80-e2c7-11ea-909a-6bccfaeaa933.png">
<img width="594" alt="Screen Shot 2020-08-20 at 9 27 47 am" src="https://user-images.githubusercontent.com/361637/90699790-8969c100-e2c7-11ea-820c-7e6b029d7641.png">

Reply:
Hi @Caleb-Ellis and @huwshimi, we're just going to replicate the legacy version for this one. No new decisions have been made.

Reply:
Thanks @amylily1011.

@Caleb-Ellis I've updated this now.